html/shared/popover0.html

```diff
@@ -0,0 +1,23 @@
+<div>
+  <h4>
+    Welcome to Paperhive
+  </h4>
+  <div class="ph-md-margin-bottom">
+    Do you like to start a 2 minutes tour now?
+  </div>
+  <div class="text-right">
+    <a class="text-muted" href="" ng-click="tour.reject();">
+      Never
```

Comment:
The links have excessive spacing at the end:

![spacing](https://cloud.githubusercontent.com/assets/181628/14102398/a3df5384-f59a-11e5-8113-dc525cefac53.png)

Also, perhaps the negative options can be moved the left-hand side of the box.